staging/src/k8s.io/cloud-provider/options/options.go

```diff
@@ -241,10 +241,3 @@ func (o *CloudControllerManagerOptions) Config(allControllers, disabledByDefault
 
 	return c, nil
 }
-
-func createRecorder(kubeClient clientset.Interface, userAgent string) record.EventRecorder {
-	eventBroadcaster := record.NewBroadcaster()
-	eventBroadcaster.StartStructuredLogging(0)
-	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
```

Comment:
do we need these too?

Reply:
we do - we we do that now as part of Run() - see lines 146-150 in controllermanager.go above